numba/cuda/tests/cudadrv/test_array_attr.py

```diff
@@ -43,22 +43,33 @@ def test_contigous_4d(self):
     def test_ravel_c(self):
         ary = np.arange(60)
         reshaped = ary.reshape(2, 5, 2, 3)
+
         expect = reshaped.ravel(order='C')
         dary = cuda.to_device(reshaped)
         dflat = dary.ravel()
         flat = dflat.copy_to_host()
         self.assertTrue(flat.ndim == 1)
         self.assertTrue(np.all(expect == flat))
 
+        # explicit order kwarg
+        for order in 'CA':
+            expect = reshaped.ravel(order=order)
+            dary = cuda.to_device(reshaped)
+            dflat = dary.ravel(order=order)
+            flat = dflat.copy_to_host()
+            self.assertTrue(flat.ndim == 1)
```

Comment:
better use `assertEqual`

Reply:
fixed all these (and some more in the same file)